InstanceProfits.lua

```diff
@@ -286,7 +292,7 @@ function IP_UpdateTime(self, elapsed)
 			triggerInstance(name, instanceDifficulty, instanceDifficultyName, true);
 		end
 		elapsedTime = 0;
-		if not enteredAlive then
+		if not enteredAlive then -- never happens
```

Comment:
This actually can happen. If the player dies in an instance, they have to reenter as a ghost to get their body back (or take spirit rez) The check for enteredAlive ensures that I don't count it as two runs every time they die, otherwise it says 1 run was from enter until death, and then a second run from death until leaving.